content/knowledge-base/binary-names-update.md

```diff
@@ -0,0 +1,18 @@
+---
+description: How to change produced binary names for Android and iOS
+title: Custom binary names
+weight: 11
+---
+
+In order to change generated the **.ipa** file name, **CFBundleDisplayName** string value in **Info.plist** needs to be renamed.If the app is configured with flavors, then **CFBundleName** string value is required to be renamed with a custom name.
```

Comment:
What is meant by 'custom name'? Can you give an example?

Reply:
By custom name, I meant that any name that a user wants to use. For example MyApp.ipa or Test.apk.